README.md

```diff
@@ -1,2 +1,31 @@
-# community
-Community related documents
+# Screwdriver Community
+
+Welcome to Screwdriver community.
+
+Check out our [docs](https://docs.screwdriver.cd/) to get information on how to [contribute](https://docs.screwdriver.cd/about/contributing/where-to-contribute), get [support](https://docs.screwdriver.cd/about/support), and see [announcements](https://docs.screwdriver.cd/about/announcements).
+
+
+## Community meetup
+
+We have public bi-weekly meeting on Mondays at 6PM PT.
+
+Where: [Google Meet](https://meet.google.com/rwo-ynwh-xqy?hs=122)
+       `Phone +1 601-844-2661 PIN: 449 924#`
+
+When:  Starting Monday, October 7 2019 between 6 - 6:30 PM PT. Repeats every 2 weeks.
```

Comment:
Should you put November 4th instead?
Looks like we don't really have to Mondays until then

Reply:
updated